gomint-server/src/main/java/io/gomint/server/network/packet/PacketPlayerlist.java

```diff
@@ -80,11 +97,12 @@ public void deserialize( PacketBuffer buffer, int protocolID ) {
         private final UUID uuid;
         private long entityId = 0;
         private String name = "";
+        private DeviceInfo deviceInfo;
         private String xboxId = "";
         private PlayerSkin skin;
 
-        public Entry( EntityHuman human ) {
-            this( human.getUUID(), human.getEntityId(), human.getPlayerListName(), human.getXboxID(), human.getSkin() );
+        public Entry( EntityPlayer player ) {
```

Comment:
Please revert this. This breaks NPC usage, if needed use a nulled DeviceInfo and use DEDICATED fallbacks for NPCs